src/utils/types.ts

```diff
@@ -10,5 +18,29 @@ export type ProfileFormInputs = {
   status: "active" | "inactive";
 };
 
+// descripes a user's public data along with their POIs
+export type PublicUser = {
+  id: string;
+  name: string | null;
+  email: string | null;
+  image: string | null;
+  bio: string;
+  preferredName: string;
+  pronouns: string;
+  role: Role;
+  status: Status;
+  seatAvail: number;
+  companyName: string;
+  startPOILocation: string;
```

Comment:
Is this referencing the start location POI name or address? Would we not want fields for both potentially?

Reply:
i don't know why we call the starting spot "startLocation" and the ending spot "companyAddress". i was under the impression that they both were just addresses...